docs/projects.md

```diff
@@ -278,3 +281,96 @@ You should see something like this:
 ```
 
 And, most importantly, your browser should open a page with a jupyter notebook.
+
+
+## Using Github
+There are three main ways to access a local copy of the Class Code, which is stored in the Github repository.  The first way is to navigate to the class Github repository click on the green code button and select 'Download Zip'.  This will provide you with a ZIP file of the class code, which can be run on your personal machine.  Now you might be saying, that is great but I don't want to download a new ZIP file every time something changes.  In that case you want a local copy of the class Github repository.  You should use on the the two following methods to accomplish this.
```

Comment:
Add commas between steps in the second sentence. Also, I believe the final sentence is meant to say "You should use one of the two following methods..."